homeassistant/components/airnow/__init__.py

```diff
@@ -47,7 +47,18 @@ async def async_setup_entry(hass: HomeAssistant, entry: ConfigEntry) -> bool:
     api_key = entry.data[CONF_API_KEY]
     latitude = entry.data[CONF_LATITUDE]
     longitude = entry.data[CONF_LONGITUDE]
-    distance = entry.data[CONF_RADIUS]
+
+    # Initialize the options parameter if the integration was first set up
+    # before OptionsFlow was added
+    if not entry.options:
+        new_data = entry.data.copy()
+        del new_data[CONF_RADIUS]
+        hass.config_entries.async_update_entry(
+            entry, data=new_data, options={CONF_RADIUS: entry.data[CONF_RADIUS]}
+        )
```

Comment:
Please do a proper [config entry migration](https://developers.home-assistant.io/docs/config_entries_config_flow_handler/#config-entry-migration)

Reply:
Does the config entry migration replace these lines then?  They were requested by @gjohansson-ST.  I don't mind making the change, but I would like to resolve conflicting review comments first so we can get this closed and committed.

Reply:
@edenhaus is right so you should implement a config entry migration and those lines above is part of that. 
See a similar PR here #94676 which is similar to what you're doing

Reply:
Thanks, that is helpful... I think I've implemented it correctly.

Reply:
Not exactly as this part is still here which should now be removed.

Reply:
Yes, silly error on my part. Fixed.